Makefile.js

```diff
@@ -130,12 +131,31 @@ function release(type) {
     echo("Publishing to git");
     exec("git push origin master --tags");
 
-    echo("Publishing to npm");
-    exec("npm publish");
+    // now push the changelog...changes to the tag
+    echo("Publishing changes to github release");
+    // this requires a github API token in process.env.ESLINT_GITHUB_TOKEN
+    // it will continue with an error message logged if not set
+    ghGot("repos/eslint/eslint/releases", {
+        body: {
+            "tag_name": newVersion,
+            name: newVersion,
+            "target_commitish": "master",
+            body: changes
+        },
+        method: "POST",
+        json: true,
+        token: process.env.ESLINT_GITHUB_TOKEN
+    }, function(pubErr) {
+        if (pubErr) {
+            echo("Warning: error when publishing changes to github release: " + pubErr.message);
+        }
+        echo("Publishing to npm");
+        exec("npm publish");
```

Comment:
Hmm... I don't think this is a good idea. By this time we already incremented version in package.json and updated CHANGELOG.md and push it to master. We should either not not push to master anything, or we should let it continue without changelog in github. At least then we can fix it manually.